contrib/linux/requirements.txt

```diff
@@ -1,3 +1,4 @@
 # used by file watcher sensor
 pyinotify>=0.9.5,<=0.10
 -e git+https://github.com/Kami/logshipper.git@stackstorm_patched#egg=logshipper
+ipaddress>=1.0.16
```

Comment:
IIRC (but I need to double check how new packages handle that) we need to also put this in global requirements.txt file.